python/dgl/graphbolt/subgraph_sampler.py

```diff
@@ -267,9 +267,16 @@ def _seeds_preprocess(minibatch):
             if use_timestamp:
                 nodes_timestamp = defaultdict(list)
             for etype, pair in seeds.items():
-                assert pair.ndim == 1 or (
-                    pair.ndim == 2 and pair.shape[1] == 2
-                ), (
+                # When pair is a one-dimensional tensor, it represents seed
+                # nodes, which does not need to do unique and compact.
+                if pair.ndim == 1:
+                    nodes_timestamp = (
+                        minibatch.timestamp
+                        if hasattr(minibatch, "timestamp")
+                        else None
+                    )
+                    return seeds, nodes_timestamp, None
+                assert pair.ndim == 2 and pair.shape[1] == 2, (
                     "Only tensor with shape 1*N and N*2 is "
                     + f"supported now, but got {pair.shape}."
                 )
```

Comment:
L284: `pair = pair.view(pair.shape[0], -1)` -> `flattened_seeds = typed_seeds.view(...)`

Reply:
We never need this line, because this line is to reshape one-dimension tensor to two-dimension one.